### ISSUE TITLE
Export as json object will be an issue with 2d files

### ISSUE THREAD

**Issue by hamed-musallam** [cheminfo/nmrium#2774]
Due to the size of the 2D data object, we are unable to stringify the entire spectra object when it becomes too large. The limitation prevents to stringify the object. To reproduce this issue, attempt to load the samples twice and then export the data as an nmrium file

https://s3.uni-jena.de/nmrxiv/production/archive/247bab07-dc3c-45cc-807d-f21bb83882f6/Genistein%20annotated%20NMR%20400%20MHz%20DMSOd6%20data.zip

**Comment by hamed-musallam** [comment]
We should consider removing the data object and migrate to file-collection

**Comment by lpatiny** [comment]
Yes it is related to https://github.com/cheminfo/nmrium/issues/2523 and this should solve the issue.
